<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>558</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>778</width>
    <height>558</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>[By 2ima] Map Editor Sample</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgba(0, 0, 0, 80);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>181</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QTreeWidget" name="w_items">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
border:  2px solid; border-bottom: none;  background-color: rgba(0, 0, 0, 80); color: rgb(105, 255, 0); 
QTreeWidget::item
{
foreground-color:green;
}


</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Elements</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Ground</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Buildings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NPCs</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="b_remove">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background: rgba(0, 0, 0, 40); border: 2px solid; </string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_add">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background: rgba(0, 0, 0, 40); border: 2px solid;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="b_loadlvl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(247, 255, 180); background: rgba(0, 0, 0, 40); border: 1.5px  solid; border-radius: 7px; </string>
   </property>
   <property name="text">
    <string>Load map</string>
   </property>
  </widget>
  <widget class="QToolButton" name="b_saveLvl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0, 0, 0, 40); border: 1.5px solid; border-radius: 7px; </string>
   </property>
   <property name="text">
    <string>Save map</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white; background: rgba(0, 100, 255, 80); border: 1.5px solid; border-radius: 9px;</string>
   </property>
   <property name="text">
    <string>Project Manager</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="w_all_items" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>541</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
